python/tvm/relay/debug_printer.py

```diff
@@ -0,0 +1,5 @@
+"""FFI exposing the Relay debug printer"""
+
+from tvm._ffi.function import _init_api
+
+_init_api("relay.debug_printer", __name__)
```

Comment:
do not need to introduce a new python file for every new function exposed, in this case we only need to expose relay.expr.DebugPrint